layouts/shortcodes/pages/community/videos.html

```diff
@@ -0,0 +1,50 @@
+<div class="card-banner card-banner-dark padding-15 margin-bottom-60">
+    <div class="row">
+        <div class="col-sm-4 match-height-item-by-row vertical-align-container">
+            <span class="featured-circle-white">
+                <img src="/images/jakarta_ee_logo_stacked.svg" alt="Jakarta EE">
+            </span>
+        </div>
+        <div class="col-sm-12 match-height-item-by-row">
+            <h2 class="big-text">Subscribe to our Youtube Channel</h2>
```

Comment:
We should probably set up these values to be i18n. I know I'm late to the party to start suggesting this, but better late then never. There was significant work to set these pages up to be compatible with language localization, and we should do our best to not take a step backwards